src/Snt/Capi/Http/ApiHttpClientConfiguration.php

```diff
@@ -39,6 +40,23 @@ public function __construct($endpoint, $apiKey, $apiSecret)
     }
 
     /**
+     * @param array $config
+     *
+     * @return ApiHttpClientConfiguration
+     * @throws InvalidArgumentException
+     */
+    public static function fromArray(array $config)
+    {
+        foreach (['endpoint', 'apiKey', 'apiSecret'] as $key) {
+            if (empty($config[$key])) {
+                throw new InvalidArgumentException(sprintf('Missing config parameter "%s"', $key));
```

Comment:
even when you name it `MissingApiHttpConfigurationArrayKeyException`? `InvalidArgumentException` might be thrown by many libraries i guess